02_configure_host.sh

```diff
@@ -81,6 +81,8 @@ ansible-playbook \
     -e "virtualbmc_base_port=$VBMC_BASE_PORT" \
     -e "master_hostname_format=$MASTER_HOSTNAME_FORMAT" \
     -e "worker_hostname_format=$WORKER_HOSTNAME_FORMAT" \
+    -e "libvirt_arch=$(uname -m)" \
+    -e "enable_vnc_console=false" \
```

Comment:
I think this will break tunnelling via SSH to the VNC console, which is very useful for debugging boot-time issues (I regularly use virt-manager on my laptop connected via `qemu+ssh` to a remote hypervisor, e.g to get info about ramdisk boot issues, or even to debug networking if you enable a user login)

I'd suggest we either remove this, or if you really want to disable vnc, add a new shell variable folks can set in their `config_$USER.sh` ?

Reply:
I'm strongly against disabling vnc_console by default, we should have a way to switch, what @hardys suggest is viable or can be triggered by architecture if this needs to be false for arm64

Reply:
I couldn't get this working on arm64 without disabling vnc. I will make it a switch that is set to false only on ARM.